database/ServiceMySql.php

```diff
@@ -81,14 +81,25 @@ public function create()
 		}
 	}
 
-	public function delete($id)
+	public function update($id)
 	{
+		try{
+			$sql = $this->conn->prepare("UPDATE guest.student SET name = :name, surname = :surname, indexno = :indexno, address = :address WHERE id = " . $id);
+
```

Comment:
what would happen if you wanted to edit only the address and leave other fields empty? :)

Reply:
It will be empty fields

Reply:
I added checking of the fields, take a look